Assets/Scripts/Model/RollModifier/ArmorModifier.cs

```diff
@@ -1,4 +1,5 @@
-﻿namespace Fumbbl.Model.RollModifier
+﻿using System.Collections.Generic;
```

Comment:
The changes to this file are reverting an updated structure. Please review this and use the existing structure instead